Bottom: 30c27a3a5de904946d2bfd883e91863a348646fd
Top:    0f7d78add9db9966b39552541f6921d89db02a22
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-23 13:37:06 +0100

Refresh of hid-png-fix-segfault-when-no-i

---

diff --git a/src/hid/png/png.c b/src/hid/png/png.c
index d2dcec4..6c0f76b 100644
--- a/src/hid/png/png.c
+++ b/src/hid/png/png.c
@@ -136,7 +136,6 @@ static int doing_outline, have_outline;
 #define FMT_png "PNG"
 
 static const char *filetypes[] = {
-  /*
 #ifdef HAVE_GDIMAGEPNG
   FMT_png,
 #endif
@@ -148,7 +147,6 @@ static const char *filetypes[] = {
 #ifdef HAVE_GDIMAGEJPEG
   FMT_jpg,
 #endif
-*/
 
   NULL
 };
